packages/cli-repl/src/cli-repl.spec.ts

```diff
@@ -1420,6 +1421,19 @@ describe('CliRepl', function () {
             expect(cliRepl.getConfig('logLocation')).is.true;
             expect(cliRepl.logWriter?.logFilePath).equals(testPath);
           });
+
+          it('can set log retention days', async function () {
```

Comment:
not sure if there are other tests that could make sense beyond getConfig / setConfig